client/graph/factory/transform.go

```diff
@@ -21,6 +21,10 @@ func EmployeeFromProtoToApi(e *proto.Employee) *model.Employee {
 	for i := range apiDocs {
 		apiDocs[i] = &e.Documents[i]
 	}
+	vacations := make([]*string, len(e.Vacations))
+	for i := range vacations {
+		vacations[i] = &e.Vacations[i]
```

Comment:
Small thing: you could use `for i,v := range vacations` and use the `v` on line 26